applications/source/mail-source/src/test/java/org/springframework/cloud/stream/app/source/file/MailSourceOldBrokenTests.java

```diff
@@ -0,0 +1,112 @@
+/*
+ * Copyright 2020-2021 the original author or authors.
```

Comment:
This is already for not `2021`...

I'm not sure in legal matter, but general rule is to update the Copyright year to the current if you make some changes to it or this is just a new class for the project at all.

Other projects may have different rules. For example I heard that Spring Data team runs a script to update all the classes in the project to a new year.
In Spring Integration we go an approach to update via Gradle when there are some local changes to classes.

So, I'd prefer to have these updated, but I don't know rules for this project.

@sobychacko ?

Reply:
Agreed, I will update and simply missed this. I update the copyrights anywhere I see them (normally).